README.md

```diff
@@ -35,7 +35,7 @@ A model of a schema with helpful methods for working with the schema and support
 The schema is initially validated (see [validate](#validate) below), and will raise an exception if not a valid JSON Table Schema.  
 
 ```javascript
-var Schema = require('jsontableschema').schema;
+var Schema = require('jsontableschema').Schema;
```

Comment:
@zhenyab 
@vitorbaptista meant here is a typo in readme (we need classes to be uppercased) but in code it was ok I suppose (Schema, Resource, infer, validate, types).